packages/logs/src/boot/startLogs.ts

```diff
@@ -30,20 +32,26 @@ export function startLogs(configuration: LogsConfiguration, errorLogger: Logger)
     trackNetworkError(configuration, errorObservable)
   }
 
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    trackConsoleLogs(['log', 'info', 'debug', 'warn'], (log) => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why errors are not collected the same way as other log levels?

Reply:
Can you clarify a bit?

Reply:
I wonder why there is no "error" in this list. It could replace the `trackConsoleError` usage.